src/main/java/svc/models/DemoCitation.java

```diff
@@ -0,0 +1,11 @@
+package svc.models;
+
+import java.time.LocalDate;
+
+public class DemoCitation {
+	public String citationNumber;
+	public LocalDate dob;
+	public String driversLicenseNumber;
+	public Integer numberOfViolations;
+	public Integer daysTillCourt;
```

Comment:
It is a demoCitation not a real citation that I return from the DemoController.
It makes life easier because this is the only data that the user needs for testing the text system.